common/views/slices/CollectionVenue/index.tsx

```diff
@@ -35,10 +38,9 @@ const CollectionVenue = ({
                 ? {
                     s: 12,
                     m: 12,
-                    l: 11,
-                    shiftL: 1,
+                    l: 12,
                     xl: 10,
-                    shiftXL: 2,
+                    shiftXL: 1,
```

Comment:
this makes perfect sense – I think it must have been a mistake that it was lop-sided before